webpack.config.js

```diff
@@ -85,13 +85,21 @@ module.exports = function makeConfig(grunt) {
         module: {
             rules: [
                 {
-                    test: /\.[j|t]sx?$/,
+                    test: /\.tsx?$/,
```

Comment:
I think it's better to use `test: /\.(ts|tsx)$/,`. It's the same thing, but easier to read